app/server.js

```diff
@@ -0,0 +1,71 @@
+const ttn = require("ttn");
+const app = require("express")();
+const http = require("http").Server(app);
+const bodyParser = require("body-parser");
+
+const googleMapsClient = require("@google/maps").createClient({
+  key: "xxxxxxxxxxxxxxxxxxxxxx"
+});
+
+var appID = "xxxxxxxxxxxxx";
+var accessKey = "ttn-account-v2.xxxxxxxxxxxxxxxxxxxxxx";
+
+var address = "";
+var wifibssids = [];
```

Comment:
Consistent naming; `wifibssids` vs `wiFiBssids` (that'll never work) vs `wifiAccessPoints`; I prefer the latter in this code